libp2p/protocols/connectivity/dcutr/client.nim

```diff
@@ -0,0 +1,63 @@
+# Nim-LibP2P
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import core
+import ../../protocol,
+       ../../../stream/connection,
+       ../../../switch
+
+import stew/results
+import chronos, chronicles
+
+type
+  DcutrClient* = ref object of RootObj
+    rttStart: Opt[Moment]
```

Comment:
Why is this here? This prevents parallel use of the system, and doesn't seem to be used outside of startSync

Reply:
True, I think it is from an early version. Removed.